hrui/hrui.php

```diff
@@ -48,18 +45,19 @@ function hrui_civicrm_pageRun($page) {
       CRM_Core_Resources::singleton()
         ->addSetting(array(
           'cid' => CRM_Utils_Request::retrieve('cid', 'Integer'),
-          'hideGId' => $hideGId));
+          'hideGId' => $hideGId)
+        );
     }
   }
 
-  CRM_Core_Resources::singleton()
-    ->addScriptFile('org.civicrm.hrui', 'js/perfect-scrollbar/perfect-scrollbar.min.js');
+  CRM_Core_Resources::singleton()->addScriptFile('org.civicrm.hrui', 'js/dist/hrui.min.js');
```

Comment:
This mean we'll get the minified version even if debug is activate, right? Do you think it's possible/necessary that we get the minified file only if debug is disabled?